content/en/docs/concepts/storage/volumes.md

```diff
@@ -14,21 +14,32 @@ weight: 10
 
 <!-- overview -->
 
-On-disk files in a container are ephemeral, which presents some problems for
+Kubernetes volumes provide a way for containers in a Pod to retain
+data beyond a container's lifecycle. They act as directories that can store data
+and are accessible across containers in the Pod.
+
+## Why volumes are important
+
+- **Data persistence:** On-disk files in a container are ephemeral, which presents some problems for
 non-trivial applications when running in containers. One problem occurs when 
-a container crashes or is stopped. Container state is not saved so all of the 
+a container crashes or is stopped, the container state is not saved so all of the 
 files that were created or modified during the lifetime of the container are lost. 
-During a crash, kubelet restarts the container with a clean state. 
-Another problem occurs when multiple containers are running in a `Pod` and 
+During a crash, kubelet restarts the container with a clean state.
```

Comment:
Volumes _do_ let you access PersistVolumes and thus get durable storage.

Reply:
Nice